src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -25,6 +27,16 @@ public WritableFileChooser(File currentDirectory) {
 	@Override
 	public void approveSelection() {
 		File selectedFile = getSelectedFile();
+		
+		if (selectedFile.getUsableSpace() < minimumSpaceRequirement) {
```

Comment:
This check needs changes, if the file does not exist the message is always shown (if it does not exist it should use the parent file instead).